spec/person_spec.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../app_code/person'
+require_relative '../app_code/rental'
+
+describe Person do
+  let(:person) { Person.new(18, 'Bambi', parents_permission: true) }
+
+  describe '#initialize' do
+    it 'creates a new Person instance with attributes' do
+      expect(person.age).to eq(18)
+      expect(person.name).to eq('Bambi')
+      expect(person.parents_permission).to be_truthy
+      expect(person.rentals).to be_empty
+      expect(person.id).to be_a(Integer)
+    end
```

Comment:
- [ ] HI @supreeti, what do you think of testing the scenario were the constructor method sets default value for the optional name parameter if it is not provided?. I think it is important to test this case :+1:

Reply:
@FrederickMih True, thanks for bringing that up, I will add that as well.